### ISSUE TITLE
API to interact with background image should be available

### ISSUE THREAD

**Issue by sguptatw** [getgauge/taiko#564]
### Expected behavior
Should be able to interact with background image using the image API

### Actual behavior
Backgound-image is not being considered as an image

### Steps to reproduce
* Execute the script 
```
const { openBrowser, goto, click, toLeftOf, image, closeBrowser } = require('taiko');
(async () => {
    try {
        await openBrowser();
        await goto("zomato.com");
        await click("Pocket-Friendly Delivery");
        await click("Order Food Online");
        await click(image(toLeftOf("Taco Bell")));
    } catch (e) {
        console.error(e);
    } finally {
        await closeBrowser();
    }
})();
```
**Notes**
The image `Get the App` is getting clicked instead of the image next to Taco Bell.
**Tech Notes**
The image next to Taco Bell is the background of a div, hence not being considered.

### Versions
```
commit 3bb2ade397840ad2326026bb7fad840161e673ea
```

**Comment by bharath-nepoleon** [comment]
We are raising a PR for this

**Comment by NivedhaSenthil** [comment]
Reopening for testing. Fix available in bc906f9

**Comment by Debashis9012** [comment]
This issue has been verified and found fixed.
Tested version: ```Version: 1.0.1 (Chromium: 76.0.3803.0) 5ada00f72c218606b9681f18966987ac606bcaba```
